internal/security/fileioperformer/methods_test.go

```diff
@@ -48,7 +48,7 @@ func TestFileIoPerformerWriter(t *testing.T) {
 
 	if isLinux() {
 		writer, err := fileio.OpenFileWriter("/dev/null", os.O_RDONLY, 0400)
-		assert.Nil(t, err)
+		assert.NoError(t, err)
```

Comment:
were these failing? Why the change?

Reply:
Better to use NoError/Error when validating error. These funcs print out the error info if they fail.

Reply:
And IMHO, it is easier to read...